src/docs/markdown/metrics.md

```diff
@@ -0,0 +1,253 @@
+if negotiated (that is, if the `Accept` header is set to
+`application/openmetrics-text; version=0.0.1`).
+
+By default, there is a `/metrics` endpoint available at the [admin API](/docs/api)
+(i.e. http://localhost:2019/metrics by default). But if the admin API is
+disabled or you wish to listen on a different port or path, you can use the
+[`metrics` handler](/docs/caddyfile/directives/metrics) to configure this.
+
+You can see the metrics with any browser or HTTP client like `curl`:
+
+```console
+$ curl http://localhost:2019/metrics
+# HELP caddy_admin_http_requests_total Counter of requests made to the Admin API's HTTP endpoints.
+# TYPE caddy_admin_http_requests_total counter
+caddy_admin_http_requests_total{code="200",handler="metrics",method="get",path="/metrics"} 2
+# HELP caddy_http_request_duration_seconds Histogram of round-trip request durations.
+# TYPE caddy_http_request_duration_seconds histogram
+caddy_http_request_duration_seconds_bucket{code="308",handler="static_response",method="GET",server="remaining_auto_https_redirects",le="0.005"} 1
+caddy_http_request_duration_seconds_bucket{code="308",handler="static_response",method="GET",server="remaining_auto_https_redirects",le="0.01"} 1
+caddy_http_request_duration_seconds_bucket{code="308",handler="static_response",method="GET",server="remaining_auto_https_redirects",le="0.025"} 1
+...
+```
+
+There are a number of metrics you'll see, that broadly fall under 3 categories:
+
+- Runtime metrics
+- Admin API metrics
+- HTTP Middleware metrics
+
+### Runtime metrics
+
+These metrics cover the internals of the Caddy process, and are provided
+automatically by the Prometheus Go Client. They are prefixed with `go_*` and
+`process_*`.
+
+Note that the `process_*` metrics are only collected on Linux and Windows.
+
+See the documentation for the [Go Collector](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewGoCollector),
+[Process Colletor](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewProcessCollector),
+and [BuildInfo Collector](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#NewBuildInfoCollector).
+
+### Admin API metrics
+
+These are metrics that help to monitor the Caddy admin API. Each of the admin
+endpoints is instrumented to track request counts and errors.
+
+These metrics are prefixed with `caddy_admin_*`.
+
+For example:
+
+```console
+$ curl -s http://localhost:2019/metrics | grep ^caddy_admin
+caddy_admin_http_requests_total{code="200",handler="admin",method="get",path="/config/"} 1
+caddy_admin_http_requests_total{code="200",handler="admin",method="get",path="/debug/pprof/"} 2
+caddy_admin_http_requests_total{code="200",handler="admin",method="get",path="/debug/pprof/cmdline"} 1
+caddy_admin_http_requests_total{code="200",handler="load",method="post",path="/load"} 1
+caddy_admin_http_requests_total{code="200",handler="metrics",method="get",path="/metrics"} 3
```

Comment:
Doesn't your latest PR in the caddyserver/caddy repo (caddyserver/caddy#3742) track methods in uppercase?

Reply:
Good point - this was copied before that fix - it's how I noticed the inconsistency 😂